Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/xdsb/helper/XDSbConstants.java

```diff
@@ -0,0 +1,471 @@
+         *
+         * @see java.lang.Enum#toString()
+         */
+        @Override
+        public String toString() {
+            return value;
+        }
+
+        /**
+         * From string.
+         *
+         * @param valueString the value string
+         * @return the xDS query status
+         */
+        public static XDSQueryStatus fromString(String valueString) {
+            if (valueString != null) {
+                for (XDSQueryStatus enumValue : XDSQueryStatus.values()) {
+                    if (valueString.equals(enumValue.toString())) {
+                        return enumValue;
+                    }
+                }
+            }
+            throw new IllegalArgumentException("No enum constant " + valueString);
+        }
+    };
+
+    /**
+     * The Enum XDSbStoredQuery.
+     */
+    public enum XDSbStoredQuery {
+
+        /**
+         * The Find documents.
+         */
+        FindDocuments("urn:uuid:14d4debf-8f97-4251-9a74-a90016b0af0d"),
+        /**
+         * The Find submission sets.
+         */
+        FindSubmissionSets("urn:uuid:f26abbcb-ac74-4422-8a30-edb644bbc1a9"),
+        /**
+         * The Find folders.
+         */
+        FindFolders("urn:uuid:f26abbcb-ac74-4422-8a30-edb644bbc1a9"),
+        /**
+         * The Get all.
+         */
+        GetAll("urn:uuid:10b545ea-725c-446d-9b95-8aeb444eddf3"),
+        /**
+         * The Get documents.
+         */
+        GetDocuments("urn:uuid:10b545ea-725c-446d-9b95-8aeb444eddf3"),
+        /**
+         * The Get folders.
+         */
+        GetFolders("urn:uuid:5737b14c-8a1a-4539-b659-e03a34a5e1e4"),
+        /**
+         * The Get associations.
+         */
+        GetAssociations("urn:uuid:a7ae438b-4bc2-4642-93e9-be891f7bb155"),
+        /**
+         * The Get documents and associations.
+         */
+        GetDocumentsAndAssociations("urn:uuid:a7ae438b-4bc2-4642-93e9-be891f7bb155"),
+        /**
+         * The Get submission sets.
+         */
+        GetSubmissionSets("urn:uuid:51224314-5390-4169-9b91-b1980040715a"),
+        /**
+         * The Get submission set and contents.
+         */
+        GetSubmissionSetAndContents("urn:uuid:e8e3cb2c-e39c-46b9-99e4-c12f57260b83"),
+        /**
+         * The Get folder and contents.
+         */
+        GetFolderAndContents("urn:uuid:b909a503-523d-4517-8acf-8e5834dfc4c7"),
+        /**
+         * The Get folders for document.
+         */
+        GetFoldersForDocument("urn:uuid:b909a503-523d-4517-8acf-8e5834dfc4c7"),
+        /**
+         * The Get related documents.
+         */
+        GetRelatedDocuments("urn:uuid:d90e5407-b356-4d91-a89f-873917b4b0e6"),
+        /**
+         * The Find documents by reference id.
+         */
+        FindDocumentsByReferenceId("urn:uuid:d90e5407-b356-4d91-a89f-873917b4b0e6");
+
+        /**
+         * The value.
+         */
+        private String value = null;
+
+        /**
+         * Instantiates a new xD sb stored query.
```

Comment:
There are a few places in the comments with "xD sb" that should be "XDSb" or "XDS.b"